### ISSUE TITLE
Instant / Vested Slider needs adjustments

### ISSUE THREAD

**Issue by BartuCulha** [PrimeDAO/prime-deals-dapp#196]

**Comment by dkent600** [comment]
Personally I find the current behavior intuitive.

**Comment by blink-km** [comment]
Agree that the current behavior is intuitive because:
1) This section is called "Instant Transfer Set-up", so the slider represents the percentage of the total amount that will be instantly transferred. When the users want to increase the ratio of instant transfer to the total amount, they move the slider to the right. In turn, the instant amount increases and the % increases.
2) This setup aligns with our visual perception where we read from left to right. So here the instant box would be the focus point where if the slider scale is at the most left point, it will show 0%. This number increases as the slider is moved towards the right side towards 100% when its at the most right point.

**Comment by giladt** [comment]
I agree with @blink-km and @dkent600 that the behavior is intuitive, but I find the **UI** is confusing.
If we'll remove the percentage away- I'd expect that if the slider is to the left- all the amount will be instant and if the slider is to the right- the whole amount will be vested.
Adding the percentage - makes it confusing.

I can suggest 3 alternatives: 

### Alternative 1
Remove vesting from the UI (or add it as simple text below)
![image](https://user-images.githubusercontent.com/2517870/155091548-bed39d2c-4839-4fa5-a1fb-0f2ed3c4982f.png)

### Alternative 2
Symetric slider: 100% <-> 50% <-> 100%:
This way the percent represents the amount of instant or vested, depending on the side it leans towards.
We can also add the word 'Instant' when it leans to the left, and 'Vested' when it leans to the right to make it even more clear.
![image](https://user-images.githubusercontent.com/2517870/155088374-59d37a58-7676-4e66-a6eb-68d4e3f0f229.png)
![image](https://user-images.githubusercontent.com/2517870/155088385-849c4cdb-5687-47a1-a987-a88cc7cf6f36.png)

### Alternative 3
Move the percentage to the labels:
![image](https://user-images.githubusercontent.com/2517870/155088817-d54d5765-9c15-4643-9593-6b04b8f0461e.png)

**Comment by blink-km** [comment]
I think **option 3** is great and addresses all the concerns and confusion. Option 1 does not provide all the required info and option 2 is more confusing. @dkent600 @BartuCulha What do you think?

**Comment by BartuCulha** [comment]
3rd option it is! Thanks @giladt and @blink-km !

**Comment by BartuCulha** [comment]
We have come to an agreement on the 3rd option recommeneded by @giladt 

We should proceed this issue by moving the percentage to the labels @dkent600 

![image](https://user-images.githubusercontent.com/24368665/155122075-58c80e79-aa11-400a-89fc-7f18b6131cb0.png)
